src/StudyEntities/Candidate/Sex.php

```diff
@@ -23,7 +23,7 @@
  * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
  * @link     https://www.github.com/aces/Loris/
  */
-class Sex
+class Sex implements \JsonSerializable
 {
     /* @var string */
     public $value;
```

Comment:
Should this be private?

Reply:
Maybe `protected`? Regardless I don't want to change something unrelated in this PR and risk introducing new bugs.